app/models/badges/boba_fett.rb

```diff
@@ -0,0 +1,19 @@
+class Badge::BobaFett
+  BADGE_ID = 7
+  DISPLAY_NAME = 'Boba Fett'
+  SLUG = 'bobafett'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.